sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -513,6 +513,12 @@ DROP TABLE lang_updates_610;
 UPDATE list_options SET notes='ms' WHERE list_id='language' AND option_id='malay';
 #EndIf
 
+#IfMissingColumn categories category_codes
+ALTER TABLE categories ADD COLUMN `codes` varchar(255) NOT NULL DEFAULT '' COMMENT 'Category codes for documents stored in this category';
+UPDATE categories SET codes='LOINC:LP173418-7' WHERE name='Advance Directive';
+UPDATE categories SET codes='LOINC:LP173421-1' WHERE name='FHIR Export Document';
+UPDATE categories SET codes='LOINC:LP173394-0' WHERE name='Reviewed';
+#EndIf
```

Comment:
would put this block at the bottom of this file (then avoid issues in ordering of the database changes)

Reply:
Interesting, the rebase must have stuck it in there, because it originally was at the bottom of the file :)